tls/s2n_connection.c

```diff
@@ -1033,11 +1033,14 @@ const char *s2n_connection_get_curve(struct s2n_connection *conn)
 {
     PTR_ENSURE_REF(conn);
 
-    if (!conn->secure.server_ecc_evp_params.negotiated_curve) {
-        return "NONE";
+    if (conn->secure.server_ecc_evp_params.negotiated_curve) {
+        /* s2n-tls currently only supports ECC groups for TLS1.3. */
```

Comment:
I think you mean "always supports". The "only supports" sounds like we don't support ECC groups for any other version.

Also, maybe "always uses" instead?

Reply:
Or maybe "TLS1.3 currently only uses"? If TLS1.3 is the subject instead of s2n-tls it's less ambiguous.

Reply:
Updated. This comment is so controversial 😦